mempool/mempool.go

```diff
@@ -191,3 +279,29 @@ func (e ErrPreCheck) Error() string {
 func IsPreCheckError(err error) bool {
 	return errors.As(err, &ErrPreCheck{})
 }
+
+// MempoolTx is a transaction that successfully ran
+type SidecarTx struct {
+	DesiredHeight int64 // height that this tx wants to be included in
+	BundleID      int64 // ordered id of bundle
+	BundleOrder   int64 // order of tx within bundle
+	BundleSize    int64 // total size of bundle
+
+	GasWanted int64    // amount of gas this tx states it will require
+	Tx        types.Tx // tx bytes
+
+	// ids of peers who've sent us this tx (as a map for quick lookups).
+	// senders: PeerID -> bool
+	Senders sync.Map
+}
+
+// Bundle stores information about a sidecar bundle
+type Bundle struct {
+	DesiredHeight int64 // height that this bundle wants to be included in
+	BundleID      int64 // ordered id of bundle
+	CurrSize      int64 // total size of bundle
+	EnforcedSize  int64 // total size of bundle
+
+	GasWanted     int64     // amount of gas this tx states it will require
```

Comment:
I do not think this is used.

Reply:
Listed in https://github.com/informalsystems/audit-skip/issues/6

Reply:
Removed in https://github.com/skip-mev/mev-tendermint/pull/131